domains/hastebin.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "EducatedSuddenBucket",
+           "email": "149485703+EducatedSuddenBucket@users.noreply.github.com",
```

Comment:
We need an actual email, not a no reply email address.

Reply:
the preview i here http://pnode3.danbot.host:3208/ and i coyuld only setup proxy after i put the records due to my host , and no reply email because of register bot , and for social there is discord

Reply:
the preview i here http://pnode3.danbot.host:3208/ and i coyuld only setup proxy after i put the records in the domain due to my host , and no reply email because of register bot , and for social there is discord

Reply:
also maskduck previosly i think in other domain merged my pr as discord was there